test/integration/call_for_evidence_test.rb

```diff
@@ -0,0 +1,143 @@
+require "test_helper"
+
+class CallForEvidenceTest < ActionDispatch::IntegrationTest
+  test "call for evidence" do
+    setup_and_visit_content_item("open_call_for_evidence")
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+
+    assert_has_metadata({
+      published: "11 April 2023",
+      last_updated: "15 May 2023",
+      from: {
+        "Office for Health Improvement and Disparities": "/government/organisations/office-for-health-improvement-and-disparities",
+      },
+    })
+
+    assert_footer_has_published_dates("Published 11 April 2023", "Last updated 15 May 2023")
+
+    within ".call-for-evidence-description" do
+      assert page.has_text?("The government is holding this call for evidence to identify opportunities to reduce the number of children (people aged under 18) accessing and using vape products, while ensuring they are still easily available as a quit aid for adult smokers.")
+    end
+  end
+
+  test "renders document attachments" do
+    setup_and_visit_content_item("closed_call_for_evidence")
+
+    assert page.has_text?("Documents")
+    within "#documents" do
+      assert page.has_text?("Net Zero Review: Call for evidence")
+    end
+  end
+
+  test "renders featured document attachments" do
+    setup_and_visit_content_item("call_for_evidence_outcome_with_featured_attachments")
+
+    assert page.has_text?("Documents")
+    within "#documents" do
+      assert page.has_text?("Setting the grade standards of new GCSEs in England – part 2")
+    end
+  end
+
+  test "link to external calls for evidence" do
+    setup_and_visit_content_item("open_call_for_evidence")
+
+    assert page.has_css?("a[href=\"#{@content_item['details']['held_on_another_website_url']}\"]", text: "another website")
+  end
+
+  test "open call for evidence" do
+    setup_and_visit_content_item("open_call_for_evidence")
+
+    assert page.has_text?("Open call for evidence")
+    assert page.has_text?(:all, "closes at 1:01pm on 16 April 2231")
+  end
+
+  test "unopened call for evidence" do
+    setup_and_visit_content_item("unopened_call_for_evidence")
+
+    assert page.has_text?("Call for evidence")
+
+    # There's no daylight savings after 2037
+    # http://timezonesjl.readthedocs.io/en/stable/faq/#far-future-zoneddatetime-with-variabletimezone
+    assert page.has_css?(".gem-c-notice", text: "This call for evidence opens at 1pm on 5 October 2201")
+    assert page.has_text?(:all, "It closes at 4pm on 31 October 2211")
```

Comment:
How come we are testing with years ~200 years into the future? Shall we rationalise them to avoid the need for comments like this? My reaction was a bit like "wtf, what's this got to do with daylight savings? then wow that's a strange year"

Reply:
I assume the intention is so that the tests don't mysteriously start to fail at some point in the future but maybe 200 years is a bit over the top! I'll need to go to tweak those in the example schemas in publishing API so they match here. Maybe 2035 would be sufficient for now?

Reply:
Sure, that'd work. I'd also wonder if we could just update the dates in the test - but maybe that's more work than needed. I'm not sure of the details to know how impractical that is or not.

Reply:
I've stubbed the date using Timecop so we can use dates around the current date without worrying about them suddenly starting to fail in the future.